dev-packages/cli/src/theia.ts

```diff
@@ -240,7 +244,12 @@ function rebuildCommand(command: string, target: ApplicationProps.Target): yargs
                     });
                 } catch (e) {
                     console.error(e);
-                    process.exit(1);
+                    exitCode = 1;
+                } finally {
+                    temp.cleanupSync();
```

Comment:
Hummm. I do not see that the cleanup is done, or at least not all the time:
```bash
ll /tmp/theia-test-config-dir2020614-
theia-test-config-dir2020614-10533-1q0vbjr.s20m/  theia-test-config-dir2020614-21303-1ldrlak.fbog/
theia-test-config-dir2020614-12184-16lblp2.1jtyf/ theia-test-config-dir2020614-22307-5tjtt5.8d7ka/
theia-test-config-dir2020614-1358-sqafxn.hn5e/    theia-test-config-dir2020614-24070-19glubk.nh63f/
theia-test-config-dir2020614-14607-1o0idz7.3iv2/  theia-test-config-dir2020614-2633-1fq9fck.8ljt/
theia-test-config-dir2020614-1532-1vm5k1j.djv8/   theia-test-config-dir2020614-5473-406jyl.z55zq/
theia-test-config-dir2020614-15375-1d76aw3.2h6lk/ theia-test-config-dir2020614-6210-1mcxy3.hsnu3/
theia-test-config-dir2020614-16444-1y5urgu.xzgx/  theia-test-config-dir2020614-7981-1n43zp4.82fo/
theia-test-config-dir2020614-17376-f2n0jo.rfz05/  theia-test-config-dir2020614-8564-uukgvl.5dopj/
theia-test-config-dir2020614-1859-f3yhxy.y6mzo/   theia-test-config-dir2020614-921-gg06ss.n8hx5/
theia-test-config-dir2020614-19147-4t9pt3.8m0t6/  theia-test-config-dir2020614-9245-xwh1m9.yv8h8/
theia-test-config-dir2020614-20182-eyk9zc.xy93e/  

```

Reply:
I've removed this line, actually `temp` library automatically call it if one called `temp.track` before. Could you try new changes? I could not reproduce it in Gitpod though, maybe `temp` library behave a bit differently on different operating system because of Node.js quirks. 🤷‍♂️

Reply:
The cleanup now works locally :+1: